From: John Zupin <johnz@lunarg.com>
Date: Thu, 30 Jul 2020 11:13:10 -0600
Subject: patch: update generated install locations with /vulkan

---
 layers/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/layers/CMakeLists.txt b/layers/CMakeLists.txt
index 323512d..5e40f42 100644
--- a/layers/CMakeLists.txt
+++ b/layers/CMakeLists.txt
@@ -72,7 +72,7 @@ if(BUILD_LAYER_SUPPORT_FILES)
         generated/vk_object_types.h
         generated/vk_extension_helper.h
         generated/vk_typemap_helper.h)
-    install(FILES ${LAYER_UTIL_FILES} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+    install(FILES ${LAYER_UTIL_FILES} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/vulkan/)
 endif()
 
 set(TARGET_NAMES
